src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double leftSide;
+    private double downSide;
+
+    public Rectangle() {
+    }
+
+    public Rectangle(double leftSide, double downSide) {
+        this.typeOfFigure = TypeOfFigure.RECTANGLE;
```

Comment:
Don't use verbs for class/interface names. The 'TypeOfFigure' enum should be named without using verbs. Consider renaming it to something like 'FigureType' to follow this convention.